recipe/meta.yaml

```diff
@@ -237,18 +238,18 @@ outputs:
         - google-pasta >=0.1.1
         - grpcio {{ libgrpc }}.*
         - h5py >=3.10
-        - ml_dtypes >={{ ml_dtypes_version }},<0.4
+        - ml_dtypes >={{ ml_dtypes_version }},<0.5
         - {{ pin_compatible('numpy', lower_bound='1.22') }}
```

Comment:
maybe remove these.

Also, seems like numpy2 might be ready in 2.17
https://github.com/tensorflow/tensorflow/issues/67291

Reply:
The release note says that "TensorFlow is going to support NumPy 2.0 in the next release."
https://github.com/tensorflow/tensorflow/blob/r2.17/RELEASE.md